pennylane/devices/null_qubit.py

```diff
@@ -129,6 +132,48 @@ def _interface(config: ExecutionConfig):
     return config.interface.get_like() if config.gradient_method == "backprop" else "numpy"
 
 
+def _simulate_resource_use(circuit):
+    RESOURCES_FNAME = "__pennylane_resources_data.json"
+
+    num_wires = len(circuit.wires)
+    gate_types = defaultdict(int)
+
+    for op in circuit.operations:
+        name = ""
+        while hasattr(op, "base"):
+            if type(op) in (Controlled, ControlledOp):
+                # Don't check this with `isinstance` to avoid unrolling ops like CNOT
+                name += "C_"
+            elif isinstance(op, Adjoint):
+                name += "adj_"
```

Comment:
To make it consistent with PL's ops names, would you be able to change this to `C(op)` and `Adj(op)`? cc: @AntonNI8

Reply:
Yes, I can do that, so for example a controlled adjoint T should render as "C(Adj(T))" ?